pandas/core/indexes/datetimes.py

```diff
@@ -1057,7 +1057,30 @@ def to_pydatetime(self):
 
     def to_period(self, freq=None):
         """
-        Cast to PeriodIndex at a particular frequency
+        Cast to PeriodIndex at a particular frequency.
+
+        Converts timestamps index to period.
```

Comment:
What do you mean by this? Oh, I think I see. How about `Converts a DatetimeIndex to a PeriodIndex`.

Reply:
Sure.I have made the changes.Thanks